experimental/singlenumberformatter/src/provider.rs

```diff
@@ -42,11 +42,11 @@ pub struct CurrencyEssentialsV1<'data> {
 
     /// Represents the standard pattern.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub standard: Cow<'data, str>,
+    pub standard: ProcessedPattern<'data>,
```

Comment:
@Manishearth , I am getting the following error:
```
error[E0277]: the trait bound `ProcessedPattern<'static>: Yokeable<'a>` is not satisfied
  --> experimental/singlenumberformatter/src/provider.rs:45:19
   |
45 |     pub standard: ProcessedPattern<'data>,
   |                   ^^^^^^^^^^^^^^^^^^^^^^^
   |                   |
   |                   the trait `Yokeable<'a>` is not implemented for `ProcessedPattern<'static>`
   |                   the trait `Yokeable<'a>` is not implemented for `ProcessedPattern<'static>`
   |
   = help: the following other types implement trait `Yokeable<'a>`:
             bool
             char
             isize
             i8
             i16
             i32
             i64
             i128
           and 24 others
```

could you help me with this error?